processes/local_development/seed_test_data.py

```diff
@@ -68,3 +70,12 @@ def save_test_record(self):
             self.db_manager.session.add(test_record)
         
         self.db_manager.session.commit()
+    
+    def save_test_data_ids_to_file(self):
+        record = self.db_manager.session.query(Record).filter_by(source_id=self.test_data['source_id']).first()
+        if record:
+            with open('test_data_ids.json', 'w') as f:
+                json.dump({
+                    'edition_id': str(record.edition_id),
+                    'uuid': str(record.uuid)
+                }, f)
```

Comment:
I like this idea to share the test ids with the tests but I think we should use fixtures to seed the data instead of the process!